### DISCUSSION TITLE
[v4.1] Redis Installation Issues

### DISCUSSION THREAD

**Discussion by joeneldeasis** [adonisjs/core#2774] [Help]
I'm currently using `v4.1` and trying to install the redis using the command found on legacy docs `adonis install @adonisjs/redis` but it does not create config file on `config/redis.js`. What was the stable version for `v4.1` and can you share also the config file for `v4.1`? I've already read #2422 but it does not help.

**Comment by thetutlage** [comment]
Install the `@legacy` tag. `adonis install @adonisjs/redis@legacy`
